src/Symfony/Component/Form/Tests/Extension/Core/Type/CheckboxTypeTest.php

```diff
@@ -170,4 +170,38 @@ public function provideCustomModelTransformerData()
             array('unchecked', false),
         );
     }
+
+    public function testSubmitNullWithEmptyDataTrueWhenNested()
+    {
+        $form = $this->factory->create('form')
```

Comment:
Oh okay, got it now what you were talking about. I changed the `CheckboxType` tests to not use nested forms and removed the nested test from the `IntegerTypeTest` where it also wasn't necessary.